packs/salt/actions/lib/base.py

```diff
@@ -1,10 +1,16 @@
-from functools import partial
-
 from st2actions.runners.pythonrunner import Action
 from requests import Request
 
 
 class SaltPackage(object):
+    _expression_forms = [
+        'glob',
+        'grain',
+        'pillar',
+        'nodegroup',
+        'list',
+        'compound'
+    ]
     _data = {"eauth": "pam",
```

Comment:
Can we also make `eauth` a configuration option?

It seems like something which is generally useful and applicable. On top of that, one of the users asked for it.